### ISSUE TITLE
Installation Badge Contrast errors

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#45698]
### Steps to reproduce the issue
1. Install Joomla and observe the contrast on the warning badges

![Image](https://github.com/user-attachments/assets/df4a471f-1115-4507-af65-f95104d4b3ea)

![Image](https://github.com/user-attachments/assets/a9e6076f-195d-4c1f-99d6-92dc10e61134)

### Expected result

No contrast errors

![Image](https://github.com/user-attachments/assets/6f9326ea-9659-4e20-aac7-2e4ba34a79b0)

### Actual result

![Image](https://github.com/user-attachments/assets/7d19b826-1743-410c-acee-7785e78c977b)

### Device and system info

* What version of Joomla were you using?
all

* What device were you using? (eg, phone, laptop)
desktop

* What web browser were you using? (eg, Chrome, Safari)
chrome

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by QuyTon** [comment]
Please test PR  #45717.
